tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+min_version = 4.0
+env_list = unitest,type,dictionaries
+
+[testenv:unitest]
+description = run unit tests
+extras = dev
+commands = pytest --cov=codespell_lib codespell_lib
+
+[testenv:type]
+description = run type checks
+extras = types
+commands = mypy codespell_lib
+
+[testenv:dictionaries]
+description = run dictionary checks
+skip_install = true
+allowlist_externals = make
+# Since pytest is not installed in this environment, we can avoid testing
+# test_dictionary.py again.
```

Comment:
I would remove this comment. The difference between test_dictionary.py and the check in teh Makefiel pre-exists. No need to document it here.

Reply:
I added the comment because I added `skip_install`.  Maybe the comment should be moved to `skip_install`, but I'm ok removing it.